modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -1114,10 +1163,9 @@ struct InferROI: public cv::detail::KernelTag {
                         // it should be treated as image
                         IE::Blob::Ptr this_blob =
                             extractBlob(*ctx, 1, cv::gapi::ie::TraitAs::IMAGE);
-                        setBlob(req,
-                                *(ctx->uu.params.input_names.begin()),
-                                IE::make_shared_blob(this_blob, toIE(this_roi)),
-                                *ctx);
+                        setROIBlob(req,
```

Comment:
I think, the function should be used for InferList and InferList2.
Can GPU ROI support appear in future?

Reply:
Do you mean - place such changes SetBlob -> SetROIBlob in other place? could you point me out, please?

Reply:
make_shared_blob for ROIs + SetBlob in InferList and InferList2. Is it necessary align this with SetRoiBlob? Now it is called in one place. 

https://github.com/opencv/opencv/blob/e81100e5e0a5bbad4dc06adb8e443db74acd9c24/modules/gapi/src/backends/ie/giebackend.cpp#L1285

https://github.com/opencv/opencv/blob/e81100e5e0a5bbad4dc06adb8e443db74acd9c24/modules/gapi/src/backends/ie/giebackend.cpp#L1444